src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient.cs

```diff
@@ -30,6 +30,7 @@ public ConfigurationClient(string connectionString, ConfigurationClientOptions o
             ParseConnectionString(connectionString, out _baseUri, out var credential, out var secret);
 
             _pipeline = HttpPipeline.Build(options,
+                    options.ResponseClassifier,
```

Comment:
I am concerned that every time we add a new concept to the pipeline we need to change the signature of the Build method. I am pretty sure we will have to add new concepts in the future, and the way "building" now works means we will have to add a new overload. Is that something that we could address?

Reply:
Yeah, I think `.Build` can just take `ResponseClassifier` of options. Not sure why I didn't do that.

Reply:
Done